src/frontend/components/split-pane/split-pane.html

```diff
@@ -0,0 +1,23 @@
+<div
+   #wrapper
+   (window:resize)="windowResized()"
+   class="flex flex-auto">
+  <div
+     #primary
+     class="col overflow-hidden flex mh100pct">
+    <ng-content select="split-pane-primary-content"></ng-content>
+  </div>
+  <div
+     #secondary
+     class="col overflow-hidden flex mh100pct">
+    <ng-content select="split-pane-secondary-content"></ng-content>
+  </div>
+  <div
+     #resizer
+     (mousedown)="resizerMouseDown($event)"
+     style="width: 6px; top: 0; bottom: 0; margin-right: -3px; cursor: ew-resize; z-index: 500; position: absolute;">
```

Comment:
None of these styles have any utility outside of their function in this module. I could spend time making little basscss nuggets for them; but it would only obscure their meaning, degrade performance, and increase the amount of code involved.

I opted for inline styles because they are functional styles.